.github/issue_template.md

```diff
@@ -0,0 +1,5 @@
+[Enter description of issue]
```

Comment:
```suggestion
Description
```

Reply:
I suggested this because it's faster to delete one word. For me the work flow would be to delete "Description" and write my description there.